### PR TITLE
Update endpoint-dlp-learn-about.md

### PR DESCRIPTION
**Pull request by omriamdursky**
Added additional file types that were missing.
Rephrased the note for default auditing that wasn't clear, I got several complaints from customers / field asking what it means, I hope this one is better.
@chrfox

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/endpoint-dlp-learn-about.md [hunk 1]
```diff
@@ -78,15 +78,17 @@ If you have onboarded devices through [Microsoft Defender Advanced Threat Protec
...
+- .c files
+- .class files
+- .cpp files
+- .cs files
+- .h files
+- .java files
 
 > [!NOTE]
-> .txt and source code files are not audited by default, DLP evaluates them against the applied policies and then user actions are audited or blocked accordingly.
+> Endpoint DLP evaluate files of all the above types against the DLP policy and applies protection actions accordingly. All files that match a DLP policy are audited for all supported actions, even if they aren't blocked. In addition, file activity performed on any Word, PowerPoint, Excel, PDF and .csv file are audited by default, indepedent of whether a DLP policy exists or matches these files.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-learn-about.md]:
```suggestion
> Endpoint DLP evaluates files of all the above types against the DLP policy and applies protection actions accordingly. All files that match a DLP policy are audited for all supported actions, even if they aren't blocked. In addition, file activity performed on any Word, PowerPoint, Excel, PDF, and .csv file is audited by default, independent of whether a DLP policy exists or matches these files.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@omriamdursky Please implement the suggested changes. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Pull request merged**